staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -57,14 +74,6 @@ func DriverName(driverName string) Option {
 	}
 }
 
-// Logger overrides the default klog.Background logger.
-func Logger(logger klog.Logger) Option {
-	return func(o *options) error {
-		o.logger = logger
-		return nil
-	}
-}
-
```

Comment:
Thank you for finally getting rid of this. I've asked ed to get rid of it a few times and he was always hesitant for some reason. I think it makes much more sense without it.

Reply:
The canonical way to do this is in Kubernetes is via contextual logging.